astroplan/scheduling.py

```diff
@@ -350,22 +559,21 @@ def from_timespan(cls, center_time, duration, **kwargs):
         duration : `~astropy.units.Quantity` or `~astropy.time.TimeDelta`
             Duration of time-span to schedule
         """
-        start_time = center_time - duration/2.
-        end_time = center_time + duration/2.
+        start_time = center_time - duration / 2.
+        end_time = center_time + duration / 2.
         return cls(start_time, end_time, **kwargs)
 
     def _make_schedule(self, blocks):
-
         # Combine individual constraints with global constraints, and
         # retrieve priorities from each block to define scheduling order
         _all_times = []
         _block_priorities = np.zeros(len(blocks))
-        for i,b in enumerate(blocks):
+        for i, b in enumerate(blocks):
             if b.constraints is None:
                 b._all_constraints = self.constraints
             else:
                 b._all_constraints = self.constraints + b.constraints
-            b._duration_offsets = u.Quantity([0*u.second, b.duration/2, b.duration])
+            b._duration_offsets = u.Quantity([0 * u.second, b.duration / 2, b.duration])
```

Comment:
I think we shouldn't hard-code the times at which the constraints are computed – maybe some users will want to compute at higher time-resolution.

You could add a `time_resolution` argument+attribute, which controls how finely to compute the constraints. We have methods for making that time grid easy to parameterize, see [`astroplan.time_grid_from_range`](https://astroplan.readthedocs.io/en/latest/api/astroplan.time_grid_from_range.html?highlight=time%20grid#astroplan.time_grid_from_range).